mne/beamformer/tests/test_lcmv.py

```diff
@@ -67,6 +67,8 @@ def _get_data(tmin=-0.1, tmax=0.15, all_forward=True, epochs=True,
     picks = mne.pick_types(raw.info, selection=left_temporal_channels)
     picks = picks[::2]  # decimate for speed
     raw.pick_channels([raw.ch_names[ii] for ii in picks])
+    raw.info['bads'] = raw.ch_names[:10]  # add more bads
```

Comment:
Did you test that things still pass locally? IIRC the LCMV/DICS tests were a bit sensitive to changing some of these params

Reply:
it runs for me locally with this change. but it would fail before.

Reply:
seems 10 bads were a bit too much for the beamformer output, it moved out of the boundaries (``AssertionError``) for testing the normal orientation.

Reply:
Furthermore, for completion: in ``test_lcmv_source_power``, ``picks`` has larger values than the number of remaining sensors.